wellregistry/registry/models.py

```diff
@@ -5,20 +5,124 @@
 from django.db import models
 from django.core.validators import MaxValueValidator, MinValueValidator
 
+class AgencyLovLookup(models.Model):
+    """Model definition for the agency_lov table, lookup only"""
+    agency_cd = models.CharField(max_length=50, primary_key=True)
+    agency_nm = models.CharField(max_length=150, blank=True, null=True)
+    org_type = models.CharField(max_length=50, blank=True, null=True)
+    agency_med = models.CharField(max_length=200, blank=True, null=True)
+    state_cd = models.CharField(max_length=5, blank=True, null=True)
+    agency_link = models.CharField(max_length=500, blank=True, null=True)
 
-class CountryLookup(models.Model):
-    """
-    Country lookup table for the Registry app.
+    class Meta:
+        db_table = 'agency_lov'
 
-    Used to populate drop downs.
+    def __str__(self):
+        return self.agency_nm
 
-    """
-    country_cd = models.CharField(max_length=10, primary_key=True)
-    country_nm = models.CharField(max_length=200)
+
+class AltDatumDimLookup(models.Model):
+    """Model definition for the alt_datum_dim table, lookup only"""
+    adatum_cd = models.CharField(max_length=10, primary_key=True)
+    adatum_desc = models.CharField(max_length=100, blank=True, null=True)
+    datum_type = models.CharField(max_length=20, blank=True, null=True)
+
+    class Meta:
+        db_table = 'alt_datum_dim'
+
+    def __str__(self):
+        return self.adatum_desc
+
+
+class CountryLookup(models.Model):
+    """Model definition for the country table, lookup only"""
+    country_cd = models.CharField(primary_key=True, max_length=2)
+    country_nm = models.CharField(max_length=48)
+
+    class Meta:
+        db_table = 'country'
 
     def __str__(self):
         return self.country_nm
 
+class CountyLookup(models.Model):
+    """Model definition for the county table, lookup only"""
+    country_cd = models.ForeignKey('CountryLookup', models.DO_NOTHING, db_column='country_cd')
+    state_cd = models.ForeignKey('StateLookup', models.DO_NOTHING, db_column='state_cd')
+    county_cd = models.CharField(max_length=3)
+    county_nm = models.CharField(max_length=48)
+    county_max_lat_va = models.CharField(max_length=11)
+    county_min_lat_va = models.CharField(max_length=11)
+    county_max_long_va = models.CharField(max_length=12)
+    county_min_long_va = models.CharField(max_length=12)
```

Comment:
just curious if latitude is always 11 chars while longitude is 12 or if just ngwmn has the difference. If it is not supposed to be different then maybe make them both 12.

Reply:
The 11 length was just copied from the NGWMN table. Good catch, it would be cleaner if they were both 12.
I'll ask if there is any reason to keep it 11.

Reply:
I think this at least in part a legacy of the NWIS site file which defines the character field for lat as 11 characters and longitude as 12 chars. Longitude's range does go from -180 to 180 whereas latitude is-90 to 90.  Not sure beyond that the merits of keeping it the way it is. I believe at least at the moment that the well registry app will not use this field.